### PR TITLE
Kubernetes Operator Release 1.1.0

### PR DESCRIPTION
**Pull request by gyfora**

### CODE REVIEW

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  name: "Gyula Fora"
+  twitter: "GyulaFora"
+- matyas:
+  name: "Matyas Orhidi"
+
+---
+
+The community has continued to work hard on improving the Flink Kubernetes Operator capabilities since our [first production ready release](https://flink.apache.org/news/2022/06/05/release-kubernetes-operator-1.0.0.html) we launched about two months ago.
+
+With the release of Flink Kubernetes Operator 1.1.0 we are proud to announce a number of exciting new features that improves the overall experience of managing Flink resources and the operator itself in production environments.
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: nit: exciting new features _improving_ the overall

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+## Release Highlights
+
+A non-exhaustive list of some of the more exciting features added in the release:
+
+ * Kubernetes Events on application and job state changes
+ * New operator metrics
+ * Unified and more robust reconciliation flow
+ * Periodic savepoint triggering
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: nit: savepoint _triggers_

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+The first version of the operator only came with basic system level metrics to monitor the JVM process.
+
+In 1.1.0 we have introduced a wide range of additional metrics related to lifecycle-management, Kubernetes API server access and the Java Operator SDK framework the operator itself is built on. These metrics allow operator administrators to get a comprehensive view of what’s happening in the environment.
+
+For details check https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/metrics-logging/#metrics.
+
+### Unified and more robust reconciliation flow
+
+We have spent a considerable effort refactoring and streamlining the core reconciliation flow responsible for executing and tracking resource upgrades, savepoints, rollbacks and other operations.
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: nit: We have _invested considerable effort into_ refactoring

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+### Periodic Savepoints
+
+By popular demand we have introduced periodic savepointing for applications and session jobs using a the following simple configuration option:
+
+```
+flinkConfiguration:
+  ...
+  kubernetes.operator.periodic.savepoint.interval: 6h
+```
+
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md:69]: Consider adding: You can also configure automatic cleanup for old savepoints triggered via this mechanism.

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+The operator allows users to listen to events and status updates triggered for the Flink Resources managed by the operator.
+
+This feature enables tighter integration with the user's own data platform. By implementing the `FlinkResourceListener` interface users can listen to both events and status updates per resource type (`FlinkDeployment` / `FlinkSessionJob`). The interface methods will be called after the respective events have been triggered by the system.
+
+### New SQL and Python Job Examples
+
+To demonstrate the power of the operator for all Flink use-cases, we have added examples showcasing how to deploy Flink SQL and Python jobs.
+
+We have also added a nice little README for the examples to make it easier for you to find what you are looking for.
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: nit: We have also added a brief README of the examples for your convenience.

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+https://github.com/apache/flink-kubernetes-operator/tree/main/examples
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: Link this from the Readme word above

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+  Normal  StatusChanged  52m   Job                   Job status changed from CREATED to RUNNING
+```
+
+### New Operator Metrics
+
+The first version of the operator only came with basic system level metrics to monitor the JVM process.
+
+In 1.1.0 we have introduced a wide range of additional metrics related to lifecycle-management, Kubernetes API server access and the Java Operator SDK framework the operator itself is built on. These metrics allow operator administrators to get a comprehensive view of what’s happening in the environment.
+
+For details check https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/metrics-logging/#metrics.
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: nit: check the list of [supported metrics](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/metrics-logging/#metrics)

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+
+The operator can watch and manage custom resources in an arbitrary list of namespaces. The watched namespaces can be defined through the property `kubernetes.operator.watched.namespaces: ns1,ns2`. The list of watched namespaces can be changed anytime in the corresponding config map, however the operator ignores the changes unless dynamic watched namespaces is enabled. This is controlled by the property `kubernetes.operator.dynamic.namespaces.enabled: true`
+
+### Experimental autoscaling support
+
+In this version we have taken the first steps toward enabling Kubernetes native autoscaling integration for the operator. The FlinkDeployment CRD now exposes the `scale` subresource which allows us to create HPA policies directly in Kubernetes that will monitor the task manager pods.
+
+This integration is still very much experimental but we are planning to build on top of this in the upcoming releases to provide a reliable scaling mechanism.
+
+https://github.com/apache/flink-kubernetes-operator/tree/main/examples#horizontal-pod-autoscaler
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: nit: Link this from the integration word in the paragraph above.

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+
+## What’s Next?
+
+We are continuing advancing on the Operator Maturity Model with an emphasis on more advanced scaling capabilities:
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: nit: _continuing our advence_

Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: What is the operator maturity model?

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+
+ * Standalone deployment mode support
+ * Hardening of rollback mechanism and stability conditions
+ * Scaling improvements
+
+## Release Resources
+
+The source artifacts and helm chart are available on the Downloads page of the Flink website. You can easily try out the new features shipped in the official 1.1.0 release by following our [quickstart guide](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-release-1.1/docs/try-flink-kubernetes-operator/quick-start/)
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: nit missing `.` at the end of paragraph

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+
+You can also find official Kubernetes Operator Docker images of the new version on [Dockerhub](https://hub.docker.com/r/apache/flink-kubernetes-operator).
+
+For more details, check the [updated documentation](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-release-1.1/) and the [release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12351723). We encourage you to download the release and share your feedback with the community through the Flink mailing lists or JIRA.
+
+## List of Contributors
+
+Aitozi, Biao Geng, Chethan, ConradJam, Dora Marsal, Gyula Fora, Hao Xin, Hector Miuler Malpica Gallegos, Jaganathan Asokan, Jeesmon Jacob, Jim Busche, Maksim Aniskov, Marton Balassi, Matyas Orhidi, Márton Balassi, Nicholas Jiang, Peng Yuan, Peter Vary, Thomas Weise, Xin Hao, Yang Wang
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: I am listed twice, remove Márton Balassi. Probably mistake on my end.

File: _posts/2022-07-25-release-kubernetes-operator-1.1.0.md [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+
+To demonstrate the power of the operator for all Flink use-cases, we have added examples showcasing how to deploy Flink SQL and Python jobs.
+
+We have also added a nice little README for the examples to make it easier for you to find what you are looking for.
+
+https://github.com/apache/flink-kubernetes-operator/tree/main/examples
+
+### Dynamic watched namespaces
+
+The operator can watch and manage custom resources in an arbitrary list of namespaces. The watched namespaces can be defined through the property `kubernetes.operator.watched.namespaces: ns1,ns2`. The list of watched namespaces can be changed anytime in the corresponding config map, however the operator ignores the changes unless dynamic watched namespaces is enabled. This is controlled by the property `kubernetes.operator.dynamic.namespaces.enabled: true`
```
Comment (mbalassi) [inline _posts/2022-07-25-release-kubernetes-operator-1.1.0.md]: nit missing . at the end of paragraph

**Review by mbalassi — COMMENTED** [review]
Built it locally, the post is properly wired to the website. Added suggestions.

**Comment by gyfora** [comment]
Thanks @mbalassi updated the post according to your suggestions :)

**Pull request closed without merging**